wp-content/themes/bzLLKit/functions.php

```diff
@@ -1059,6 +1059,8 @@ function bz_attendance($atts, $content = null) {
 
     return "<iframe onload=\"
         this.style.height = (30 + this.contentWindow.document.body.scrollHeight) + 'px';
+        // fall back to a minimum height because the iframe document height trick doesn't work in IE
+        this.style.minHeight = '500px';
```

Comment:
I'm slightly curious how this would look when using windowed mode that is smaller than `500px`. Have you checked that?

Reply:
Also, have you checked this with other browsers?

Reply:
I've *only* checked it with other browsers. 😉 If we like this solution I'll get this on staging and ask Traci to test it since she has IE. 

Not sure what you mean on the windowed mode question.

Reply:
There is a bug in IE where min-height doesn't calculate properly under certain conditions, but I can't remember how to exactly cause it. We can merge it in and I can test it on staging with one of my computers.